resources/assets/js/components/attendance/AttendanceKiosk.vue

```diff
@@ -43,7 +43,9 @@ export default {
           if (rawTeams.length < 1) {
             swal('Bueller...Bueller...', 'No teams found.', 'warning');
           } else {
-            this.teams = response.data.teams.sort(function(a, b) {
+            this.teams = response.data.teams.filter(function(item) {
+              return item.visible && item.attendable;
```

Comment:
Only check for `attendable` - we may want to have teams that appear on the kiosk but not on the website (training maybe???)

Reply:
Training definitely per bossman